### DISCUSSION TITLE
Well, seems like Os.js is dead.

### DISCUSSION THREAD

**Discussion by nequals** [os-js/OS.js#832] [General]
R.I.P. OS.js.
You will be remembered.

**Comment by andersevenrud** [comment]
Officially? I don't remember sending out that memo.

**Reply by ajmeese7** [reply]
Apparently because we have lives OUTSIDE of free software, we've killed one of the great projects of the internet! We're terrible people, @andersevenrud.

**Reply by nequals** [reply]
I don't imply having lives outside of FOSS software is bad. And I don't blame anyone for not updating os.js. All I say is it was not recieving any updates for, well, almost half a year.

**Reply by andersevenrud** [reply]
> recieving any updates for, well, almost half a year.

That doesn't mean its dead :) Also, FYI, if you look at the repository list then you'll see last update was last month.

This (`OS.js`) repo very rarely gets any updates because it does not contain any base source code. All of that is provided with npm packages :)

**Comment by nequals** [comment]
K imma go fork this
